### ISSUE TITLE
Tables in doc

### ISSUE THREAD

**Issue by cchantep** [playframework/playframework.com#3]
Seems styles for table inside page body can be improved for doc generated from markdown on playframework.com (e.g. type table at end of Anorm page).

```css
#content sup {
  font-size: x-small; 
  vertical-align:top }

#content table tr {
  background-color: #fff;
  border-top: 1px solid #ccc
}

#content table th, #content table td {
  border: 1px solid #ddd;
  padding: 6px 13px
}

#content table th { font-weight: bold }

#content table tr:nth-child(2n) { background-color: #f8f8f8 }
```

**Comment by wsargent** [comment]
So the bug is that https://www.playframework.com/documentation/2.5.x/Anorm has tables but doesn't look so hot.

**Comment by cchantep** [comment]
Currently such table are not readable in the Play doc.

**Comment by wsargent** [comment]
https://github.com/playframework/anorm/blob/master/docs/manual/working/scalaGuide/main/sql/ScalaAnorm.md

**Comment by wsargent** [comment]
Wait, this is https://github.com/playframework/playframework/blob/2.5.x/documentation/manual/releases/release24/migration24/Anorm.md

**Comment by wsargent** [comment]
Fixed by #57
